src/main/python/serialization_classes.py

```diff
@@ -18,6 +18,18 @@ def __init__(self, parammod):
         self._articulators = parammod.articulators
         self.timingintervals = parammod.timingintervals
         self._addedinfo = parammod.addedinfo
+        self._phonlocs = parammod.phonlocs
+
+    @property
```

Comment:
@kvesik I was inspired by your latest PR. lmk if this looks good to you : )

Reply:
@Harukaichii yes! But I think this will also let you get rid of the `hasattr(serialmodule, 'phonlocs')` checks in `Sign.unserializehandconfigmodules()` and `Sign.unserializenonmanualmodules` ?

Reply:
@kvesik not quite. But still doable if I moved a check here : https://github.com/PhonologicalCorpusTools/SLPAA/pull/350/files#diff-4399ecb482ab4e1509aa0b611083119ec240f9122265e6131e779e238a0411c0R133